html-forms/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="GET">
+      <div>
+        Email:
+        <div><input type="text" name="email"></div>
+        Why is coding cool?
+        <div>
+          <select>
+            <option value="select">Select an option</option>
+            <option value="interesting">Because it's interesting</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </div>
+        Favorite Lanuage (choose one)
+        <div>
+          <input type="radio" name="lang" value="html">HTML
+          <input type="radio" name="lang" value="css">CSS
+          <input type="radio" name="lang" value="javascript">JavaScript
+        </div>
+        Would you like to sign up for more information?
+        <div>
+          <input type="checkbox" name="subscribe"> Sign me up!
+        </div>
+        Comments
```

Comment:
you should put this in a `label` element. Make sure you set up a properly configured `for` attribute!